bundle/manifests/extensions.istio.io_wasmplugins.yaml

```diff
@@ -133,14 +133,14 @@ spec:
                       type: string
                       x-kubernetes-validations:
                       - message: wildcard not allowed in label value match
-                        rule: '!self.contains("*")'
+                        rule: '!self.contains(''*'')'
```

Comment:
Just curious, why escaping single quote instead of using double quotes?

Reply:
this is from the upstream CRD. no idea honestly, this is usually all generated

Reply:
It goes through a tool so its all mechanical

Reply:
Thank you